tests/HtmlMatcherTest.php

```diff
@@ -78,14 +78,12 @@ public function considersValidHtml_WHtmlContainsScriptTagWithHtmlContents()
     public function addsSpecificTextInsideTheSciptTagsInsteadOfItsContents()
     {
         $html = "<div>
-<script type='x-template'>
-	<span></span>
-</script>
+<script type='x-template'><span></span></script>
 </div>";
 
         assertThat($html, is(htmlPiece(havingChild(
             both(withTagName('script'))
-                ->andAlso(havingTextContents(HtmlMatcher::SCRIPT_BODY_REPLACEMENT))))));
+                ->andAlso(havingTextContents("<span><\\/span>"))))));
```

Comment:
It's the same problem I've already noted at mediawiki phpcs Phabricator board:
```php
// shortest
$prefixed = array_map(function ($i) { return 'x' . $i; }, $array);

// mediawiki style
$prefixed = array_map(function ($i) { 
    return 'x' . $i; 
}, $array);

// consistent in terms of argument list wrapping
$prefixed = array_map(
    function ($i) { 
        return 'x' . $i; 
    }, 
    $array
);
```

The last is the most consistent in terms of formatting, the first is the shortest one, the middle is what is used in mediawiki style.
As soon as there is no complex stuff happening in closure (as in example) and it doesn't hit width limit, I would prefer the first style because it is readable and "concentrated".
I think this kind of decision is up to a human and should not be enforced by style checker.
As well as this case, for example:
```php
assertThat($html, is(htmlPiece(havingChild(
    both(withTagName('script'))
        ->andAlso(havingTextContents("<span><\\/span>"))))));
// VS more consistent
assertThat(
    $html,
    is(
        htmlPiece(
            havingChild(
                both(withTagName('script'))->andAlso(havingTextContents("<span><\\/span>"))
            )
        )
    )
);
```